slices/iproute2.yaml

```diff
@@ -0,0 +1,75 @@
+package: iproute2
+
+slices:
+  # iproute2 is a utility package which typically could be split into
+  # utility-specific slices. The `bins` slice however brings in all the
+  # utilities for cases where iproute2 is used as a dependency for another
+  # package (eg. nginx), where we can't easily know which utilities are
+  # actually going to be needed.
+  bins:
+    # Leaving the debconf dependency out as we don't currently need pkg mgmt.
+    # Also, debconf is used by the pkg's maintainer scripts, which are handled
+    # differently in Chisel. 
+    essential:
+      - iproute2_config
+      - iproute2_libs
+      - libcap2-bin_bins
+    contents:
+      /bin/ip:
+      /sbin/ip:
+      /bin/ss:
+      /sbin/bridge:
+      /sbin/dcb:
+      /sbin/devlink:
+      /sbin/rtacct:
+      /sbin/rtmon:
+      # "tc" is an obvious candidate for getting its own "traffic-control"
+      # slice in the future, if needed.
+      /sbin/tc:
+      /sbin/tipc:
+      /sbin/vdpa:
+      /usr/bin/lnstat:
+      /usr/bin/ctstat:
+      /usr/bin/rtstat:
+      /usr/bin/nstat:
+      /usr/bin/rdma:
+      /usr/bin/routel:
+      /usr/sbin/arpd:
+      /usr/sbin/genl:
```

Comment:
nit: I understand the reason behind keeping file and it's symlinks together. But I think it would be good for the future to keep the file list sorted.

Reply:
Y I thought about this. I haven't yet convinced myself of what's the best approach. Nonetheless, and given that many other SDFs have a similar structure, I recommend we take this into consideration when writing the contribution guidelines (which is something that might go alongside the new CI we're planning to have)